src/preset.ts

```diff
@@ -878,13 +878,13 @@ async function expandBuildPresetHelper(preset: BuildPreset,
     if (util.isString(preset.targets)) {
       preset.targets = await expandString(preset.targets, expansionOpts);
     } else {
-      for (const index in preset.targets) {
+      for (let index = 0; index < preset.targets?.length; index++) {
```

Comment:
else if (preset.targets), and remove all the `?`s in this change.

Reply:
actually it's already checked (hidden in compare view...). Can you remove all the '?'s in your changes?

Reply:
Yes. I noticed the same. We've already validated the types of the presets file at this point, right?  I don't need to make sure it's an array or anything do I?

Reply:
That's correct.